<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DBresetClass</class>
 <widget class="QMainWindow" name="DBresetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>393</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>499</width>
    <height>393</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>499</width>
    <height>393</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DBreset</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btn_clear">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>230</y>
      <width>171</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>All Clear</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>461</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>281</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_serverPath">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_serverPath">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>100</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>461</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Select Database Type</string>
    </property>
    <widget class="QRadioButton" name="radio_sqlite">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>86</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Sqlite DB</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_web">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>86</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Web DB</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="edit_dataPath">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>140</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_dataPath">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>140</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Database Folder:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Image Folder:</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="DBreset.qrc"/>
 </resources>
 <connections/>
</ui>
